com.ugetdm.uGet.yaml

```diff
@@ -0,0 +1,119 @@
+app-id: com.ugetdm.uGet
+runtime: org.gnome.Platform
```

Comment:
does this really need the GNOME runtime?

Reply:
Fails on Freedesktop
```
Package 'libnotify', required by 'virtual:world', not found
```

Reply:
libnotify is fairly wasy to build and will also be in freedesktop runtime 23.08.

If that's the only thing, can you try including libnotify here and switch to freedesktop runtime?

Reply:
gstreamer also I think

Reply:
it's also in freedesktop.

Reply:
Switched to freedesktop